recipes/pycld2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pycld2" %}
+{% set version = "0.41" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycld2-{{ version }}.tar.gz
+  sha256: a42f6e974df8fdd70685c2baa8a9f523069a260e1140ce604fb9f1fb6c3064df
+
+build:
+  number: 0
+  noarch: python
```

Comment:
```suggestion
```

Reply:
🐑 How did I not catch that?

Reply:
I removed that, but still can't be able to build